ui/scripts/main.js

```diff
@@ -3,4 +3,11 @@ var main = document.getElementById("container");
 for(i=0; i<data['game_data'].length; i++) {
     var stage_data = data['game_data'][i];
     main.innerHTML += Handlebars.templates.stage({data: stage_data});
+
+    // add three empty boxes because Level 1 has 3 stages instead of 6
+    if (stage_data.stage === '1-3' && data['game_data'].length > 3) {
```

Comment:
This is totally fine. A nicer way to do it is if we split the string at the - and check the first number. Makes no difference but the code in that case tests exactly what we want